docs/README.md

```diff
@@ -0,0 +1,319 @@
+
+
+---
+
+## ☁️ Gridlook collections
+
+**🧊 DKRZ support for S3 and Swift datasets**
+
+STAC Integration as a DM test service:
+Buttons as Assets in STAC Items
+
+- [Dyamond3 Healpix](https://discover.dkrz.de/external/stac2.cloud.dkrz.de/fastapi/collections/dyamond)
+
+----
+
+**🧊 DKRZ support for S3 and Swift datasets**
+
+works well for the HK25 data stored at DKRZ because of
+
+1.
+
+**a performant institutional s3 cloud storage**
+
+----
+
+**🧊 DKRZ test support for S3 and Swift datasets**
+
+works well for the HK25 data stored at DKRZ because we
+
+2. 
+ 
+**prepared datasets**
+
+- rechunked datasets
+- enriched datasets (`crs`, attributes)
+- brought it to cloud
+- embedded links in catalogs
+
+----
+
+![Traffic](assets/traffic.png)
+
+----
+
+### ☁️ Other Cloud Providers
+
+For CORS-enabled locations:
+
+💡 Right-click `.zmetadata` to get the URL
+→ append to Gridlook URL and remove `.zmetadata`
+→ submit
+
+----
+
+- **Google Cloud**  
+  [CMIP6 Dataset](https://console.cloud.google.com/marketplace/product/noaa-public/cmip6)
+
+- **AWS**  
+  [EURO-CORDEX Dataset](https://registry.opendata.aws/euro-cordex/)
+
+- (**Azure**  
+  ❌ Not supported (e.g., [Planetary Computer](https://planetarycomputer.microsoft.com/dataset/gridmet)))
+
+---
+
+## BYODataset
+
+*A minimal guide for DKRZ users:*
+
+1. Make your dataset [CF](https://cfconventions.org/) conform for gridtype identification. Add
+
+    - `grid_mapping` attribute used for *"rotated_lat_lon"* and *"healpix"*
+    - coordinates: `lat` and `lon` values
+    - better not encode `time` as *"INT64"*
+    - (`long_name` attribute to variables)
+
+----
+
+2. Store zarr datasets in DKRZ cloud storage: [swift](https://docs.dkrz.de/doc/datastorage/swift/index.html) (until 2026) and [s3](https://docs.dkrz.de/doc/datastorage/minio/index.html) soon.
+
+```shell
+cdo -f nc4c copy INPUT OUTPUT #reformat to zarr
+module load swift #login
+swift upload BUCKET OUTPUT
+```
+
+----
+
+Without temporary output in Python with [swiftspec](https://github.com/fsspec/swiftspec)
+
+```python
+import xarray as xr
+ds = xr.open_dataset(YOUR_DS_URI)
+ds.to_zarr("swift://BUCKET/DS_NAME")
+```
+
+Example: [Fsspec intro](https://github.com/eerie-project/EERIE_hackathon_2023/blob/main/nereus/tutorial_cloud_fsspec.ipynb)(EERIE Hackathon 2023)
+
+----
+
+3. Bucket setting: Publish and allow CORS
+
+```bash
+swift post BUCKET -r .r:* #publish 
+swift post BUCKET -m \
+  "X-Container-Meta-Access-Control-Allow-Origin:*" #set CORS
+```
+
+----
+
+**Optimal Zarr-datasets for Web-Apps**:
+
+Reduce the amount of transferred data.
+
+- Chunks!
+  - aim for 1-5 MB
+  - are loaded in parallel (it's ok to load a few)
+  - are *cached* (on client-side usually only if smaller than 50MB)
+
+- Compress it!
+  - [zarrita supports common algorithms](https://github.com/manzt/zarrita.js/blob/c0dd684dc4da79a6f42ab2a591246947bde8d143/packages/zarrita/src/codecs.ts#L26)
+
+
+---
```

Comment:
Everything after that should be deleted. "Outlook" is something for the issue tracker

Reply:
I disagree. With a date in the header, status should be clear.